apps/src/Sound.js

```diff
@@ -339,22 +340,61 @@ Sound.prototype.getPlayableFile = function() {
   return false;
 };
 
+Sound.prototype.getPlayableBytes = function() {
+  try {
+    if (!window.Audio) {
+      return false;
+    }
+
+    var audioTest = new window.Audio();
+    if (
+      this.config.hasOwnProperty('bytes') &&
+      audioTest.canPlayType('audio/mp3')
+    ) {
+      return this.config.bytes;
+    }
+  } catch (e) {
+    console.warn('No bytes provided or mp3 is not supported');
+  }
+
+  return false;
+};
+
 Sound.prototype.preload = function() {
   var file = this.getPlayableFile();
-  if (!file) {
+  var bytes = this.getPlayableBytes();
+  if (!file && !bytes) {
     return;
   }
 
   if (!this.config.forceHTML5 && window.AudioContext && this.audioContext) {
     var self = this;
-    this.preloadViaWebAudio(file, function(buffer) {
-      self.reusableBuffer = buffer;
-    });
+    if (file) {
+      this.preloadViaWebAudio(file, function(buffer) {
+        self.reusableBuffer = buffer;
+      });
+    } else {
+      self.audioContext.decodeAudioData(bytes, function(buffer) {
+        self.reusableBuffer = buffer;
+        // THIS IS FOR FIREFOX
```

Comment:
Nit: Can you give a little more context? i.e. I've seen similar workarounds before written like this:
"Workaround for Firefox: In Firefox, there is a specific..."